spec/features/admin/sort_view_in_organisation_list_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe 'sorting the columns in the organisation list' do
+  context 'when admin views the table with organisation data' do
+    let!(:user) { create(:user, :confirmed, admin: true) }
+
+    before do
+      create(:organisation, name: "Department of Apple Cakes", created_at: Date.yesterday)
+      create(:organisation, name: "Department of Silly Hats", created_at: 1.month.ago)
+      create(:organisation, name: "Department of Xylophones", created_at: 2.years.from_now)
+
+      sign_in_user user
```

Comment:
I think this file needs `require 'features/support/sign_up_helpers'` at the top, as that's where this `sign_in_user` method is defined. 

On my machine, this spec file fails if I run it in isolation.